packages/extension/src/newtab/App.tsx

```diff
@@ -106,6 +89,7 @@ function InternalApp(): ReactElement {
   const isPageReady =
     (growthbook?.ready && router?.isReady && isAuthReady) || isTesting;
   const shouldRedirectOnboarding = !user && isPageReady && !isTesting;
+  const isFirefox = process.env.TARGET_BROWSER === 'firefox';
```

Comment:
We should probably globally export isFirefoxExtension and maybe make it more explicit.
We only target firefox extension, reading this might make you think otherwise.